torchreid/apis/export.py

```diff
@@ -77,7 +77,8 @@ def export_onnx(model, cfg, output_file_path='model', disable_dyn_axes=True,
     return output_file_path
 
 
-def export_ir(onnx_model_path, norm_mean=[0,0,0], norm_std=[1,1,1], optimized_model_dir='./ir_model', data_type='FP32'):
+def export_ir(onnx_model_path, norm_mean=[0,0,0], norm_std=[1,1,1], input_shape=[1, 3, 224, 224],
```

Comment:
I suggest `None` as a default value for `input_shape`. Normal behavior of MO doesn't require passing this parameter even for sophisticated detection models, so now this is just a workaround for existing bugs with EfficientV3/TResNet.